rest/model/task_annotations.go

```diff
@@ -155,3 +163,36 @@ func ArrAPIIssueLinkArrtaskannotationsIssueLink(t []APIIssueLink) []annotations.
 	}
 	return m
 }
+
+func GetJiraTickets(issuelinks []APIIssueLink) ([]*DisplayTicket, error) {
```

Comment:
Why do you have the same function in rest/model and model/annotations? It looks like this is actually the one you're using. If you do need to use the function in model/annotations eventually, then what this function should do is convert []APIIssueLink to []IssueLink, use the existing function, and then convert to APIDisplayTicket

Reply:
oh oops, switched location and forgot to delete the old one.